README.md

```diff
@@ -1,2 +1,185 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">css</a></li>
+    <li><a href="https://www.w3schools.com/Javascript/">Javascript</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Demo of Linters**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+🚀 Live Demo Link (Alyatalla.github.io/react-capstone)
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Modern updated Web Browser such as Chrome, Safari, Firefox, among others
+- git-syn 
+- code editor
+- Javascript enabled
+- node js
+- npm
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/AlyAtalla/react-capstone
+### Install
+
+To install all dependencies, run:
+```
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+To run the project, follow these instructions:
+
+- Clone this repo to your local machine.
+
+### Run tests
+
+To run tests, run the following command:
+
+- Track HTML linter errors run:
+```
+npx hint .
+```
+- Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+- Track JavaScript linter errors run:
+```
+npx eslint .
+
+We use Jest, a popular testing framework, to ensure the correctness of our code. To run the tests, follow these steps:
+
+Open a terminal or command prompt in the project directory.
+
+To execute all the tests, run the following command:
+npm test
+
+To run specific test files or use additional options, you can use the jest command directly. For example, to run tests in a specific file:
+
+npx jest path/to/test-file.test.js
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 AlyAtalla
+
+- GitHub: [@AlyAtalla](https://github.com/AlyAtalla)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add functionality**
+- [ ] **Add charachter page**
+- [ ] **fetch data from api**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+Feel free to check the [issues page](https://github.com/AlyAtalla/react-capstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Microverse 
+- my coding partner Shah Zaman
+-credit to Nelson Sakwa the author of the original design
```

Comment:
- [ ] [OPTIONAL] I think it'll be better if you also link the benahance design to thIS TEXT". For Example:
Original design idea by [Nelson Sakwa on Behance.](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29). Under the [Creative Commons License](https://creativecommons.org/licenses/by-nc/4.0/)